index.bs

```diff
@@ -632,19 +632,281 @@ Issue(12): Propose control protocol for Remote
+states that are available cross-origin depending on the user's network
+context.  Exposure of this data to the Web is also discussed in
+[[PRESENTATION-API#personally-identifiable-information]] and
+[[REMOTE-PLAYBACK#personally-identifiable-information]].
+
+### Origin Access to Other Devices ### {#origin-access-devices}
+
+By design, the Open Screen Protocol allows access to presentation displays and
+remote playback devices from the Web.  By implementing the protocol, these
+devices are knowingly making themselves available to the Web and should be
+designed accordingly.
+
+Below, we discuss mitigation steps to prevent malicious use of these devices.
+
+### Incognito Mode ### {#incognito-mode}
+
+The Open Screen Protocol does not distinguish between the user agent's normal
+browsing and incognito modes, and should operate identically regardless of which
+mode is in use.
+
+It's recommended that controllers use separate authentication contexts and QUIC
+connections for normal and incognito profiles from the same user agent instance.
+This prevents Open Screen receivers from correlating activity between a user's
+normal profile and their incognito profile.
+
+### Persistent State ### {#persistent-state}
+
+An implementation of the Open Screen Protocol is likely to persist the identity
+of controllers and receivers that have successfully completed
+[[#authentication]].  This may include the public key fingerprints, metadata
+versions, and metadata for those parties.
+
+However, this data is not normally exposed to the Web, only through the native
+UI of the user agent during the display selection or display authentication
+process.  It can be an implementation choice whether the user agent clears or
+retains this data when the user clears browsing data.
+
+Issue(132): [Privacy] Fate of metadata / authentication history when clearing
+browsing data
+
+### Other Considerations ### {#other-considerations}
+
+The Open Screen Protocol does not grant to the Web additional access to the
+following:
+
+* New script loading mechanisms
+* Access to the user's location
+* Access to device sensors
+* Access to the user's local computing environment
+* Control over the user agent's native UI
+* Security characteristics of the user agent
+
+Presentation API Considerations {#presentation-api-considerations}
 -------------------------------
 
-Mitigations and defenses {#security-defenses}
----------------------------------------------
+[[PRESENTATION-API#security-and-privacy-considerations]] place these
+requirements on the Open Screen Protocol:
+
+1.  Presentation URLs and presentation IDs should remain private among the
+    parties that are allowed to connect to a presentation, per the
+    cross-origin access guidelines.
+1.  Controllers and recievers should be notified when multiple connections have
+    been made to a presentation, per the user interface guidelines.
+1.  Messaging between presentations and controllers should be authenticated and
```

Comment:
Do you mean between receivers and controllers?

Reply:
Fixed